### DISCUSSION TITLE
Interference in footage

### DISCUSSION THREAD

**Discussion by jackl123** [blakeblackshear/frigate#2836] [Ask A Question]
Lately, my Frigate footage has intermittently been picking up blocky shape artefacts (for want of a better term - see attached screenshots to see what I mean) which aren't on the original camera stream. Does anyone have any idea of what could be causing this and/or what I can do to prevent it please?

I'm running HA on a Raspberry Pi 4, and wonder if it's a sign of the hardware struggling or something else.

<img width="551" alt="Screenshot 2022-02-19 at 22 01 28" src="https://user-images.githubusercontent.com/13586561/154820671-5c377d33-da0b-41b6-8489-eafa499b82bd.png">

<img width="318" alt="Screenshot 2022-02-19 at 22 03 26" src="https://user-images.githubusercontent.com/13586561/154820649-926bcd4a-8793-452f-b95f-501e6cd067d3.png">
.

**Comment by blakeblackshear** [comment]
Typically I see this when a camera uses UDP rather than TCP and there is packet loss. Both of those examples could have been eliminated with a `min_area` filter for person.

**Reply by jackl123** [reply]
Got it, thanks for your reply

**Reply by ManuelPerrot** [reply]
I had similar issue with reolink camera and rstp stream.
I have swapped to http and it seems to have gone away.
https://docs.frigate.video/configuration/camera_specific/
